doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+
+   The command should return ``+CFUN: 4``, which indicates that the modem is in offline state.
+   If it returns a different value, repeat the previous step.
+#. Click :guilabel:`Certificate manager` in the navigation bar to switch to the certificate manager view.
+
+   .. figure:: /images/ltelinkmonitor_navigationcertificatemanager.png
+      :alt: LTE Link Monitor - Certificate Manager
+
+      LTE Link Monitor - Certificate Manager
+
+#. Click :guilabel:`Load from JSON` and select the :file:`*.cert.json` file tat you downloaded from nRF Cloud.
+   Alternatively, you can drag and drop the file onto the GUI.
+
+   .. figure:: /images/ltelinkmonitor_loadjson.png
+      :alt: LTE Link Monitor - Load from JSON
+
+      LTE Link Monitor - Load from JSON
+
+#. Ensure that the :guilabel:`Security tag` is set to ``16842753``, which is the security tag for nRF Cloud credentials.
+#. Click :guilabel:`Update certificate`.
+
+   .. figure:: /images/ltelinkmonitor_updatecertificates.png
+      :alt: LTE Link Monitor - Update certificates
+
+      LTE Link Monitor - Update certificates
+
+   The log message "Certificate update completed" indicates that the certificate was provisioned successfully.
+   If you encounter any errors, switch to the terminal view and check the output of the AT commands that were sent to the nRF9160 DK modem.
+
+   .. note::
+
+      If you had connected your nRF9160 DK to nRF Cloud before, you must delete the device there after provisioning the certificate.
+      To do so, open the entry for your device from the :guilabel:`Devices` view, then click the gear icon to the right of the device's name, and select :guilabel:`Delete Device`.
+      Then, add the nRF9160 DK again as described in :ref:`nrf9160_gs_connecting_dk_to_cloud`
+
+.. _nrf9160_gs_connecting_dk_to_cloud:
+
+Connecting the DK to nRF Cloud
+******************************
+
+To transmit data from your nRF9160 DK to nRF cloud, you must activate your SIM card and add the nRF9160 DK to your nRF Cloud account.
+
+Complete the following steps:
+
+1. Go to `nRF Cloud`_ and sign in.
+#. Click the :guilabel:`+` icon in the top left corner.
+   The :guilabel:`Add New` window appears.
+
+   .. figure:: /images/nrfcloud_plus_sign_callout.png
+      :alt: nRF Cloud - Plus icon
+
+      nRF Cloud - Plus icon
+
+#. Click :guilabel:`LTE Device`.
+   The :guilabel:`Activate SIM Card` window appears.
+
+   .. figure:: /images/nrfcloud_add_lte_device1.png
+      :alt: nRF Cloud - Add New window
+
+      nRF Cloud - Add New window
+
+#. Activate your iBasis SIM card by entering the 18-digit :term:`Integrated Circuit Card Identifier (ICCID)` and the :term:`Personal Unblocking Key (PUK)`, accepting the terms and the privacy policy, and clicking :guilabel:`Activate SIM`.
+
+   .. figure:: /images/nrfcloud_activating_sim.png
+      :alt: nRF Cloud - Activate SIM Card window
+
+      nRF Cloud - Activate SIM Card window
+
+   The ICCID and the PUK are printed on the SIM card.
```

Comment:
You need to scratch off the back of the SIM card before you enter the codes so this should come earlier. The image showing the locations of the codes on the SIM card is missing.

Reply:
Maybe the steps in Thingy:91 could be reused here.

Reply:
The image is out of date with currently shipping SIM cards, the location of the scratch off has moved to the bottom of the card and there is a bar code with another code at the top.

Added mention of scratching off the back.